server/src/@common/filestore/cloud.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { appConfig } from 'config/config';
+import { FileStoreUtilsService } from './utils.service';
+import { IFileStoreCloudFile } from './fileStore.interface';
+import { S3Client, PutObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+
+let s3Client: S3Client;
```

Comment:
Does this need to be outside the class or can it be in the FileStoreCloudService constructor as `this.s3Client = new S3Client(...` ?

Reply:
Yeah good catch, fixed